projects/native-charts-ng/src/public-api.module.ts

```diff
@@ -4,11 +4,12 @@
  */
 import { NgModule } from '@angular/core';
 
+import { SiMicroDonutComponent } from './components/si-micro-donut/si-micro-donut.component';
 import { SiNChartGaugeComponent } from './components/si-nchart-gauge/si-nchart-gauge.component';
 
 @NgModule({
-  imports: [SiNChartGaugeComponent],
-  exports: [SiNChartGaugeComponent]
+  imports: [SiNChartGaugeComponent, SiMicroDonutComponent],
+  exports: [SiNChartGaugeComponent, SiMicroDonutComponent]
```

Comment:
This blows up everyone's bundle, Lets just enforce apps to directly import the component